ispyb-update-all-proposals

```diff
@@ -28,42 +31,26 @@ def main(args=[]):
         else:
             logging.basicConfig()  # "WARNING" by default
 
-    cycle = args.cycle
     dry_run = args.dry_run
 
-    all_cycles = nsls2api_lib.get_all_cycles()
-    all_proposals = set()
-    for cycle in all_cycles:
-        props = nsls2api_lib.get_proposals_from_cycle(cycle["name"])
-        try:
-            props_list = props[0]["proposals"]
-            #print(f"{cycle['name']}: {sorted(props_list)}")
-            for proposal in props_list:
-                all_proposals.add(proposal)
-        except KeyError as e:
-            print(f"KeyError - most likely empty proposals list. continuing. cycle: {cycle['name']} error: {e}")
-    print(f"Starting with a total of {len(all_proposals)} proposals")
-    ispyb_proposals = set()
-    for proposal in sorted(all_proposals):
-        proposal_info = nsls2api_lib.get_proposal_info(proposal)
-        if "error_message" in proposal_info.keys():
-            continue  # no useful proposal info - such as a missing one
-        if nsls2api_lib.is_ispyb_instrument(proposal_info["instruments"]):
-            print(f"adding {proposal_info['proposal_id']}")
-            ispyb_proposals.add(proposal_info["proposal_id"])
+    ispyb_proposals = nsls2api_lib.get_all_proposal_ids_for_instruments(
```

Comment:
I don't see this function in the https://github.com/NSLS2/nsls2api-client-for-mx/pull/2/files pr, am I looking in the wrong spot?